vsock_test.go

```diff
@@ -44,3 +48,45 @@ func TestAddr_fileName(t *testing.T) {
 		})
 	}
 }
+
+func TestUnblockAcceptAfterClose(t *testing.T) {
+	listener, err := Listen(1024)
+	if err != nil {
+		t.Fatalf("failed to run listener: %v", err)
+	}
+
+	var wg sync.WaitGroup
+
+	wg.Add(1)
+
+	go func() {
+		defer wg.Done()
+
+		t.Log("start accept")
+		_, err := listener.Accept()
+		t.Log("after accept")
+		if err != nil {
+			return
```

Comment:
Do something in this block?  Maybe panic if you get an error other than one caused by the socket closing?

Reply:
Done